### PR TITLE
Replace static CSS class with a data attribute on Preview

### LINKED ISSUE #334: Define a separate attribute for accessing user rendered components
Right now class names are misused for that purpose.  The reasoning is - classes used internally for styling should be a private implementation detail of sg, they shouldn't be used at the same time for accessing containers for [visual testing](https://github.com/styleguidist/react-styleguidist/issues/238). It makes a false assumption for a user that he should override styles using class names.  It is better to use the default generateClassName function which generates unique class names, to make the intent clear.

In order to give a user an access to the container through the dom, we can use for e.g. a 'data-container' attribute with a specified container name or even path.

### PR DESCRIPTION
**Pull request by n1313**
Fixes #334

### CODE REVIEW

File: src/rsg-components/Playground/PlaygroundRenderer.js [hunk 1]
```diff
@@ -75,7 +74,7 @@ export function PlaygroundRenderer({
 }) {
 	return (
 		<div className={classes.root}>
-			<div className={cx(classes.preview, 'rsg--example-preview')}>
+			<div className={classes.preview} data-preview={name}>
 				<div className={classes.isolatedLink}>
 					{name && (
```
Comment (sapegin) [inline src/rsg-components/Playground/PlaygroundRenderer.js]: Looks like `name` can be empty but I’m not sure why. Maybe we can remove this condition ;-)

Comment (n1313) [inline src/rsg-components/Playground/PlaygroundRenderer.js]: Maybe we can, but that's out of scope of this PR :)

Comment (sapegin) [inline src/rsg-components/Playground/PlaygroundRenderer.js]: Then you need to handle case when name is not there ;-)

Comment (n1313) [inline src/rsg-components/Playground/PlaygroundRenderer.js]: Ah, now I see what you are talking about. OK!

**Comment by n1313** [comment]
If `name` is not defined then data-preview attribute will be present but empty.

**Comment by sapegin** [comment]
Thanks!

**Pull request merged**